index.html

```diff
@@ -1350,6 +1348,115 @@ <h3>alsoKnownAs</h3>
         </div>
 
       </section>
+
+      <section>
+        <h3>sameAs</h3>
+          <p>
+            A <a>DID Method</a> can define different forms of a DID that are 
+            logically equivalent. An example is when a DID takes one form prior 
+            to registration in a verifiable data registry and another form after 
+            such registration. In this case, the <a>DID Method</a> specification 
+            may need to express one or more DIDs that are logically equivalent to 
+            the resolved DID as a property of the DID document. This is the 
+            purpose of the sameAs property.
+          </p>
+  
+          <dl>
+            <dt><dfn>sameAs</dfn></dt>
+            <dd>
+              The value of <code>sameAs</code> MUST be a 
+              <a data-cite="INFRA#lists">list</a> where each item in the list 
+              is a DID conforming to the requirements in Section 3.1.
+            </dd>
+            <dd>
+              The relationship is a statement that each <code><a>sameAs</a></code> 
+              DID value is logically equivalent to the <code>id</code> property DID 
+              value and thus identifies the same DID subject.
+            </dd>
+            <dd>
+              Each <code><a>sameAs</a></code> DID value MUST be produced by the same 
+              DID method as the <code>id</code> property DID value.
+            </dd>
+            <dd>
+              A conforming <a>DID Method</a> specification MUST guarantee that each 
+              <code><a>sameAs</a></code> DID value is logically equivalent to the 
+              <code>id</code> property DID value.
+            </dd>
+            <dd>
+              A resolving party SHOULD retain the DIDs from the <code>id</code> and 
```

Comment:
I think this should is not testable, how can I write a test that proves an RP stores 2+ identifiers instead of 1?